syntax = "proto3";

option java_multiple_files = true;
option java_package = "lis.gcv.genesearch";
option java_outer_classname = "GeneSearchProto";
option objc_class_prefix = "GS";

package genesearch;


service GeneSearch {
  rpc Search (SearchRequest) returns (SearchReply) {}
}


message SearchRequest {
  string query = 1;
}


message SearchReply {
  repeated string genes = 1;
}
